### ISSUE TITLE
Add local database support so that the feature for observing the real time database change without using fetch query 

### ISSUE THREAD

**Issue by neeluagrawal04** [jonataslaw/getx#2361]
**ATTENTION: DO NOT USE THIS FIELD TO ASK SUPPORT QUESTIONS. USE THE PLATFORM CHANNELS FOR THIS. THIS SPACE IS DEDICATED ONLY FOR FEATURE REQUESTS**

**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is.
I want to fetch the data from database but by the time query run and get the data from database, the data is not filled in database and the query will return 0 records. So there should be a mechanism where i can observe the database changes and reflect on ui without fetching query till the data records return greater than 0.

**Describe the solution you'd like**
On App launch I want to get the list of users from server and insert it to database. By the time user reaches to home screen of app and if the data is not inserted in database the fetch query on home screen return 0 records and I need to query again to fetch the data from database. So in order to avoid using the fetch query again and again till the records count is greater than 0. There should be a live data observer which observe changes in database and update Ui accordingly.

**Describe alternatives you've considered**
using stream
fetching data again

**Comment by Shreemanarjun** [comment]
I don't think this is a getx problem. You may have to change your db client (hive ro sqflite) to sqlbrite or any other db that supports reactive stream.

**Comment by neeluagrawal04** [comment]
How to use the reactive stream via floor database in order to fetch the data with Getx widegt?
